### DISCUSSION TITLE
Add a config option to the enchant command to block incompaitable enchants.

### DISCUSSION THREAD

**Discussion by IDerSArl97** [EssentialsX/Essentials#4434] [Ideas and Feature Suggestions] [answered]
Currently, the /enchant command only checks if the requested enchant can be applied to the item and is "safe" when configured to. It does not check if the enchant being requested is allowed on the item as what does the Minecraft version of the command/anvils do, thus allowing combinations such as fortune and silk touch to be applied to an item.

**Comment by JRoy** [comment, marked as answer]
We already use Enchantment#canEnchantItem to validate this.
